apps/main-app/src/components/dataTable/AnnotatedEntitiesPopupCuration.js

```diff
@@ -66,18 +78,17 @@ function AnnotatedEntitiesPopupCuration(props) {
                 entities.map(entity => (
                   <tr key={entity.subject.curie}>
                     <td>{renderLink(entity)}</td>
-                    <td>
-                      <TypeCellCuration subject={entity.subject}/>
-                    </td>
-                    <td>
-                      <ExperimentalConditionCellCuration conditions={entity.conditionRelations} />
-                    </td>
-                    <td>
-                      <ExperimentalConditionCellCuration conditions={entity.conditionModifiers} />
-                    </td>
-                    <td>{entity.singleReference &&
-                      SingleReferenceCellCuration(entity.singleReference)
-                    }</td>
+                    <td><TypeCellCuration subject={entity.subject}/></td>
+                    <td>{entity.assertedGenes && AssertedGenes(entity.assertedGenes)}</td>
+                    <td><ExperimentalConditionCellCuration conditions={entity.conditionRelations}/></td>
+                    <td><ExperimentalConditionCellCuration conditions={entity.conditionModifiers}/></td>
+                    <td>{entity.sgdStrainBackground && StrainBackground(entity.sgdStrainBackground)}</td>
+                    <td>{entity.geneticSex && (entity.geneticSex.name ? entity.geneticSex.name: '')}</td>
+                    <td>{entity.relatedNotes && RelatedNotes(entity.relatedNotes)}</td>
+                    <td>{entity.annotationType && (entity.annotationType.name ? entity.annotationType.name: '')}</td>
+                    <td>{entity.evidenceCodes && <EvidenceCodesCellCuration evidenceCodes={entity.evidenceCodes}/>}</td>
+                    <td>{entity.dataProvider && (entity.dataProvider.abbreviation ?? '')}</td>
```

Comment:
I think it would be better to turn all these into components. And then you can do the logic inside the component rather then here.

Reply:
For instance: `<td>{entity.relatedNotes && RelatedNotes(entity.relatedNotes)}</td>` turns into:

`<td>{RelatedNotes(entity.relatedNotes)}</td>` and do the logic checking inside the component.

Reply:
Moved, but I have been not able to test because staging is not returning data for genes